src/components/logo/Witchcraft.astro

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
I actually really like the setup plain Astro components give you. Just html/css/js as it should be. Opted for here so that I can include the color-changeing script tag without needing to re-hydrate a Vue component.